src/Analyzers/CSharp/Analyzers/CSharpAnalyzers.projitems

```diff
@@ -83,6 +83,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)UseCollectionExpression\CSharpUseCollectionExpressionForEmptyDiagnosticAnalyzer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UseCollectionExpression\CSharpUseCollectionExpressionForArrayDiagnosticAnalyzer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UseCollectionExpression\UseCollectionExpressionHelpers.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)UseCollectionInitializer\CSharpUseCollectionInitializerAnalyzer.cs" />
```

Comment:
there used to be a single `UseCollectionInitializerAnalyzer` type that was used for both VB and C#.  HOwever, it was getting crushed under the weight of having to have special logic (esp. logic that was really only for C#'s collection expressions).  

We now use inheritance here, which cleans up a few things (especially around type arguments).